src/test/java/com/virtuslab/archunit/BackgroundTaskEnqueuingTestSuite.java

```diff
@@ -58,7 +58,7 @@ private boolean isTaskBackgroundableQueue(JavaMethodCall call) {
         })
         .because("otherwise it's likely that you forgot about actually scheduling the task; " +
             "mark the method with ${BackgroundableQueuedElsewhere.class.getSimpleName()} if this is expected")
-        .check(importedClasses);
```

Comment:
`imported` didn't really bring any meaning: if the classes is visible to ArchUnit, it's surely been imported somehow. `production`, however, states that the checks aren't executed against test classes.